### PR TITLE
Add Friction tutorial

### PR DESCRIPTION
**Pull request by sathvikbhagavan**

### CODE REVIEW

File: docs/src/friction.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+# Modeling Non Linear Friction Model using UDEs
+
+Friction between moving bodies is not trivial to model. There have been idealised linear models which are not always useful in complicated systems. There have been many theories and non linear models which we can use, but they are not perfect. The aim of this tutorial to use Universal Differential Equations to showcase how we can embed a neural network to learn an unknown non linear friction model.
+
+## Julia Environment
+
+First, lets import the required packages.
```
Comment (Vaibhavdixit02) [inline docs/src/friction.md:7]:
```suggestion
First, let's import the required packages.
```

And multiple places below, it just caught my eye sorry 😅

File: docs/src/friction.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+## Model Setup
+
+Now, lets learn the same friction model using a neural network. We will use [`NeuralNetworkBlock`](@ref) to define neural network as a component. The input of the neural network is the velocity and the output is the friction force. We connect the neural network with the model using `RealInput` and `RealOutput` blocks.
+
+```@example friction
+function friction_ude(Fu)
+    @variables y(t) = 0.0
+    @constants Fu = Fu
+    @named nn_in = UDEComponents.RealInput2(nin = 1)
+    @named nn_out = UDEComponents.RealOutput2(nout = 1)
```
Comment (Vaibhavdixit02) [inline docs/src/friction.md:84]: Are these functions going to be public API?

File: docs/src/friction.md (hunk 1 above)
Comment (sathvikbhagavan) [reply docs/src/friction.md:7]: Thanks! Will clean it up.

File: docs/src/friction.md (hunk 2 above)
Comment (sathvikbhagavan) [reply docs/src/friction.md:84]: No, I am waiting on https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/283, so for the time being I had hardcoded it to try out the training.

File: src/ModelingToolkitNeuralNets.jl [hunk 3]
```diff
@@ -1,7 +1,8 @@
 module ModelingToolkitNeuralNets
 
-using ModelingToolkit: @parameters, @named, ODESystem, t_nounits
-using ModelingToolkitStandardLibrary.Blocks: RealInput, RealOutput
+using ModelingToolkit: @parameters, @named, ODESystem, t_nounits, @connector, @variables,
+                       Equation
```
Comment (SebastianM-C) [inline src/ModelingToolkitNeuralNets.jl:4]: where are `@connector` and `Equation` used?

Comment (sathvikbhagavan) [reply src/ModelingToolkitNeuralNets.jl]: They should have been removed. Removed it in [37890ff](https://github.com/SciML/ModelingToolkitNeuralNets.jl/pull/19/commits/37890ff0ff65c3f2ee9136bbc0d7af2bcbb099bf)

File: docs/src/friction.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+Let's plot it.
+
+```@example friction
+scatter(sol_ref, label = "velocity")
+```
+
+That was the velocity. Let's also plot the friction force acting on the object throughout the simulation.
+
+```@example friction
+scatter(sol_ref.t, friction.(first.(sol_ref.u)), label = "friction force")
```
Comment (SebastianM-C) [inline docs/src/friction.md:73]: This could use the `idxs` kwarg.

**Pull request merged**